components/resource/resourcesystem.hpp

```diff
@@ -41,6 +41,9 @@ namespace Resource
         /// @note May be called from any thread if you do not add or remove resource managers at that point.
         void updateCache(double referenceTime);
 
+        /// Indicates to each resource manager to clear the entire cache.
```

Comment:
Could use a comment on thread safety, since we're already doing that for the other methods.